gateway/src/apicast/policy/rate_limit/rate_limit.lua

```diff
@@ -10,6 +10,10 @@ local limit_traffic = require "resty.limit.traffic"
 local ts = require ('apicast.threescale_utils')
 local tonumber = tonumber
 local next = next
+local format = string.format
+local insert = table.insert
+local os_date = os.date
```

Comment:
Better to use `ngx.time` or `ngx.now` as they are way more performant in scope of OpenResty.